templates/app/controllers/store_controller.rb

```diff
@@ -5,6 +5,8 @@ class StoreController < Spree::BaseController
   include Spree::Core::ControllerHelpers::Order
   include SolidusStarterFrontend::Taxonomies
 
+  layout 'application'
```

Comment:
I think I considered naming the layout differently, though I ultimately went with `application` since that's the default layout that comes with Rails. That being said, if we name the layout `storefront`, we can avoid deleting any application layout already present in the app. I think the less we force changes on the app, the more robust the template will be. What do you think?

Reply:
> I think the less we force changes on the app, the more robust the template will be. What do you think?

I agree!

Reply:
Okay, I'll change the layout to `storefront`. TODO.